README.md

```diff
@@ -177,6 +177,181 @@ ApolloConverterFactory apolloConverterFactory = new ApolloConverterFactory.Build
         .build();
 ```
 
+## Support For Cached Responses
+
+Apollo GraphQL client allows you to cache responses, making it suitable for use even while offline. The client can be configured with 3 levels of caching:
+
+ - **HTTP Response Cache**: For caching raw http responses. These responses are stored in files.
+ - **Normalized Disk Cache**: Per node caching of responses in SQL. Persists normalized responses on disk so that they can used across sessions. 
+ - **Normalized InMemory Cache**: Optimized Guava memory cache for when cached normalized responses need not be persisted across sessions.  
+
+### Usage
+
+Raw HTTP Response Cache:
+```java
+//Directory where cached responses will be stored
+File file = new File("/cache/");
+
+//Size in bytes of the cache
+int size = 1024*1024;
+
+//Strategy for deciding when the cache becomes stale
+EvictionStrategy evictionStrategy = new TimeoutEvictionStrategy(5, TimeUnit.SECONDS);
+
+//Create the http response cache store
+ResponseCacheStore cacheStore = new DiskLruCacheStore(file, size); 
+
+//Build the Apollo Client
+ApolloClient apolloClient = ApolloClient.builder()
+                                    .serverUrl("/")
+                                    .httpCache(cacheStore, evictionStrategy)
+                                    .okHttpClient(okHttpClient)
+                                    .build();
+```
+
+Normalized Disk Cache:
+```java
+//Create the ApolloSqlHelper
+ApolloSqlHelper apolloSqlHelper = ApolloSqlHelper.create(context, "db_name");
```

Comment:
"Please note `null` is passed in as the name, you will get an in-memory SqlLite database that will not persist across restarts of the app"

This is sqllite not us fyi